src/main/java/bio/terra/workspace/service/get/GetService.java

```diff
@@ -0,0 +1,49 @@
+package bio.terra.workspace.service.get;
+
+import bio.terra.workspace.app.configuration.ApplicationConfiguration;
+import bio.terra.workspace.common.exception.SamApiException;
+import bio.terra.workspace.common.utils.SamUtils;
+import bio.terra.workspace.db.WorkspaceDao;
+import bio.terra.workspace.generated.model.CreatedWorkspace;
+import bio.terra.workspace.generated.model.GetWorkspaceRequestBody;
+import bio.terra.workspace.service.iam.AuthenticatedUserRequest;
+import bio.terra.workspace.service.iam.SamService;
+import org.broadinstitute.dsde.workbench.client.sam.ApiException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GetService {
+  private ApplicationConfiguration configuration;
+  private final WorkspaceDao workspaceDao;
+  private final SamService samService;
+
+  @Autowired
+  public GetService(
+      ApplicationConfiguration configuration, WorkspaceDao workspaceDao, SamService samService) {
+    this.configuration = configuration;
+    this.workspaceDao = workspaceDao;
+    this.samService = samService;
+  }
+
+  public CreatedWorkspace getWorkspace(
```

Comment:
Need to eventually return a more descriptive representation of the workspace but for now there's not much to a workspace. I could also wire through the spend profile and policies, I suppose...